sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+2. SIG has three co-chairs, who are TOC Contributors and recognised as unbiased experts in that area.
+
+3. SIG has one TOC liaison who is a voting member of the TOC acting as non-exec chair on occasions when TOC input is deemed necessary by the TOC or the SIG chairs
+
+4. SIG has multiple tech leads who are recognised as unbiased experts in the SIG area, and are recognised leaders of projects in the SIG’s area.  The reason for having separate chair and tech lead roles is to allow responsibility for primarily administrative functions to be separated from deep technical functions and associated time commitments and skill sets.  Where appropriate, an individual may perform both roles (see below).
+
+5. Thought and interest diversity is strongly encouraged within SIGs.  To this end, a supermajority (⅔ or more) of chairs or tech leads from a single group of companies, market segment, etc will be actively discouraged by the TOC.
+
+6. SIG members are self-declared, so that some SIG work is done by volunteers from the TOC Contributors and community.  To recognise members who make sustained  and valuable contributions to a SIG over time, SIG-defined and assigned roles may be created (e.g. scribe, training or documentation coordinator etc).  SIG’s should document what these roles and responsibilities are, and who performs them, and have them approved by SIG leads.
+
+### SIG Member Roles
+
+#### Chair
+
+* Three chairs where the active chair rotates each week/fortnight/month.
+
+* Primarily performs administrative functions including collecting and compiling topics for the (bi)weekly agenda, chairing the meeting, ensuring that quality meeting minutes are published, and follow-up actions tracked and resolved.
+
+* A chair role may be held and performed by a tech lead, in cases where one person has the time and ability to perform both roles to the satisfaction of the TOC and SIG members.
+
+#### Tech Lead
+
+* Leads projects in the SIG’s area.
+
+* Has the time and ability to perform deep technical dives on projects.   Projects may include formal CNCF projects or other projects in the area covered by the SIG.
+
+#### Other named roles
+
+* Named and defined by the SIG (e.g. scribe, PR lead, docs/training lead, etc)
+
+* Approved by supermajority of the chairs.
+
+#### Other members
+
+* Self-declared
+
+* May either have no explicit roles or responsibilities, or formally assigned roles (see above).
+
+* May not create the impression that they have any authority or formal responsibilities in the SIG other than assigned roles.
+
+### Elections
+
+* The TOC nominates Chairs
+
+* Chairs are assigned following a 2/3 majority vote of the TOC
+
+* Terms last for 1 year but staggered such that at least 1 of the chairs is able to maintain continuity
+
+* The TOC and Chairs nominate Tech leads
+
+* Tech leads are assigned following a majority vote of the TOC and SIG Chairs
```

Comment:
2/3 majority or just majority?  You seem to use 2/3 in other places.

Reply:
We should be clear on whether 50% is is a majority or not for cases where there are an even # of voters. Also, be clear that people who abstain from voting are not marked as "no" votes - meaning, the tally is based on just those people who voted not over all people that can vote.  This is true for the entire doc.

Reply:
Clarified that 2/3 majority is required.  Detailed voting mechanics are covered in the charter.  The intention is to follow those established decision-making protocols, rather than define new ones.